movieapp/src/components/movielist/error-boundary.tsx

```diff
@@ -0,0 +1,11 @@
+type ErrorProps = {
+  isOk: boolean;
+  children: JSX.Element;
+};
+
+function ErrorBoundary(props: ErrorProps): JSX.Element {
```

Comment:
it's not error boundary.

check it https://reactjs.org/docs/error-boundaries.html

Reply:
fixed